l10n_it_vat_statement_split_payment/tests/test_vat_statement_split.py

```diff
@@ -0,0 +1,286 @@
+        ).invoice_date
+
+        cls.account_tax_22sp = cls.company_data["default_tax_sale"].copy(
+            {
+                "name": "22% SP",
+                "amount": 22,
+                "amount_type": "percent",
+                "vat_statement_account_id": received_vat_account,
+                "type_tax_use": "sale",
+            }
+        )
+
+        cls.account_tax_22 = cls.company_data["default_tax_sale"].copy(
+            {
+                "name": "22%",
+                "amount": 22,
+                "amount_type": "percent",
+                "vat_statement_account_id": received_vat_account,
+                "type_tax_use": "sale",
+            }
+        )
+        cls.account_tax_22_credit = cls.company_data["default_tax_purchase"].copy(
+            {
+                "name": "22% credit",
+                "amount": 22,
+                "amount_type": "percent",
+                "vat_statement_account_id": paid_vat_account,
+                "type_tax_use": "purchase",
+            }
+        )
+
+        cls.sp_fp = cls.fp_model.create(
+            {
+                "name": "Split payment",
+                "split_payment": True,
+                "tax_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "tax_src_id": cls.account_tax_22.id,
+                            "tax_dest_id": cls.account_tax_22sp.id,
+                        },
+                    )
+                ],
+            }
+        )
+        cls.company = cls.company_data["company"]
+        cls.company.sp_account_id = cls.env["account.account"].search(
+            [
+                (
+                    "account_type",
+                    "=",
+                    "asset_current",
+                )
+            ],
+            limit=1,
+        )
+        cls.a_recv = cls.account_model.create(
+            dict(
+                code="cust.acc",
+                name="customer account",
+                account_type="asset_receivable",
+                reconcile=True,
+            )
+        )
+        cls.a_sale = cls.env["account.account"].search(
+            [
+                (
+                    "account_type",
+                    "=",
+                    "income",
+                )
+            ],
+            limit=1,
+        )
+        cls.vat_authority = cls.account_model.create(
+            {
+                "code": "VAT.AUTH",
+                "name": "VAT Authority",
+                "reconcile": True,
+                "account_type": "liability_payable",
+            }
+        )
+
+        cls.account_payment_term = cls.term_model.create(
+            {
+                "name": "16 Days End of Month",
+                "note": "16 Days End of Month",
+            }
+        )
+        cls.term_line_model.create(
+            {
+                "value": "balance",
+                "days": 16,
```

Comment:
Non ci andrebbe anche
```suggestion
                "days": 16,
                "end_month": True,
```
?